chevah/github_hooks_server/tests/test_handler.py

```diff
@@ -794,8 +773,8 @@ def test_getTicketFromTitle(self):
         self.assertIsNone(ticket_id)
 
         self.assertLog(
-            b'Pull request has no ticket id in title: '
-            b'Simle words 12 r\xc3\x89sume.')
+            'Pull request has no ticket id in title: '
+            'Simle words 12 r\xc9sume.')
```

Comment:
For convenience, I weakened this test; it only checks the string itself, no longer the byte representation.
Should I decode the string as UTF-8 bytes, and compare them?